tests/ui/absurd-extreme-comparisons.rs

```diff
@@ -12,27 +12,46 @@ fn main() {
     const Z: u32 = 0;
     let u: u32 = 42;
     u <= 0;
+    //~^ ERROR: this comparison involving the minimum or maximum element for this type con
```

Comment:
If these could use `//~` instead for ones that are a single error then that'd be great, would require tons of manual work though...

Reply:
I'd prefer to keep having the message level. The goal is to ensure both that the message content and the message level don't get updated unnoticed.

Reply:
I mean having it inline, not omitting the level, like
```rust
    u <= 0; //~ ERROR: etc
```

Reply:
Oh sorry, misunderstood. So yes, a lot of manual work which I don't have time for, sorry about that... Do you mind if we keep it this way?

Reply:
I don't, it's fine as is. Maybe we can update the script you used at some point to do this instead